test/e2e/framework.go

```diff
@@ -637,6 +637,19 @@ func (data *TestData) mutateFlowAggregatorConfigMap(ipfixCollector string, faClu
 	return nil
 }
 
+// deleteFlowAggregator deploys flow aggregator with ipfix collector address.
+func (data *TestData) deleteFlowAggregator() error {
```

Comment:
I am curious as to why we delete the YAML and re-apply it instead of deleting the Flow Aggregator Pod and letting K8s restart it?

Reply:
It will be very quick for the Flow Aggregator to restart when deleting the Pod, but the default interval for checking whether the connection is open is 10 seconds. Flow Exporter may not catch the error and automatically reconnect to the Service again in that way because the cluster IP remains the same.

Reply:
Updated with deleting the Flow Aggregator Pod. Thanks!